lib/msf/base/sessions/mettle_config.rb

```diff
@@ -18,12 +17,22 @@ def initialize(info = {})
               'MeterpreterTryToFork',
               'Fork a new process if the functionality is available',
               default: false
-            )
+            ),
           ]
         )
+        unless staged?
+          register_advanced_options(
+            [
+              OptEnum.new(
+                'PayloadLinuxMinKernel',
+                [true, 'Linux minimum kernel version for compatibility', '2.6+', ['2.6+', '3.17+']]
```

Comment:
Presumably `+` would be implied here. Making users specify the `+` seems unnecessary.

A minimum version is a minimum version (`2.6`), not a version range (`2.6+`).